src/CMakeLists.txt

```diff
@@ -214,7 +214,7 @@ set(MICROPY_COMPILER_FLAGS
 
 	${BSP_CPU_FLAGS}
 	)
-
+add_compile_options(${COMPILE_FLAGS})
```

Comment:
This was added so that it would be seen globally by the app's CMake file.
Otherwise all of the static libraries in bm_core would have to have `target_compile_options` for the 6 static libraries being built